Tests/Provider/AbstractProviderTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\Provider;
```

Comment:
@Soullivaneuh Indeed, I didn't intend for this class to be public... Should I mark it as `@internal` or something else? Thanks.

@core23 There is no `Sonata\MediaBundle\Test` namespace. Did you mean `Sonata\MediaBundle\Tests` ? I also don't understand why to move it ? Thanks